app/models/user.rb

```diff
@@ -3,4 +3,15 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  enum role: [:user, :moderator, :admin]
+  after_initialize :set_default_role, :if => :new_record?
```

Comment:
Mozna to usunąć

Ogolnie after callbacków rzadko sie uzywa, wprowadzają dużo zamieszania, możemy defaulta ogarnąć po stronie bazy danych, zobacz jak nowa migracja wygląda

Wtedy automatycznie jak będzie się tworzył user to baza przypisze mu defaulta